workspaces/src/network/sandbox.rs

```diff
@@ -19,7 +19,7 @@ use crate::types::{AccountId, Balance, InMemorySigner, SecretKey};
 use crate::{Account, Contract, Network, Worker};
 
 // Constant taken from nearcore crate to avoid dependency
-pub(crate) const NEAR_BASE: Balance = 1_000_000_000_000_000_000_000_000;
+pub const NEAR_BASE: Balance = 1_000_000_000_000_000_000_000_000;
```

Comment:
This is ugly const that I would prefer not to expose as part of the public interface. https://github.com/near/near-token will hopefully emerge next week and that would be a better way to deal with NEAR tokens as a type

Reply:
It won't be exposed because the `sandbox` module is private.

Reply:
This is a false positive: https://github.com/rust-lang/rust-clippy/issues/8319